bucket/rakudo-star.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://rakudo.org/",
+    "version": "2020.01-01",
+    "license": "Artistic-2.0-Perl",
+    "description": "The most mature, production-ready implementation of the Raku language",
+    "url": "https://rakudo.org/dl/star/rakudo-star-2020.01-01-win-x86_64-(JIT).msi",
+    "hash": "A4A728FFF7C32D98790E4E12F145BFAB89CC4DAB56E268B2CED3C80F08BD4A30",
+    "extract_dir": "rakudo",
+        "bin": [
+        "bin\\perl6.exe",
+        "bin\\raku.exe",
+        "bin\\rakudo.exe"
+    ],
+    "checkver": {
+        "url": "https://rakudo.org/downloads/star",
+        "regex": "rakudo-star-([\\d]{4}.[\\d]{2}-[\\d]{2})-win-x86_64-\\(JIT\\).msi"
+    },
+    "autoupdate": {
+        "url": "https://rakudo.org/latest/star/win"
```

Comment:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://rakudo.org/dl/star/rakudo-star-$version-win-x86_64-(JIT).msi"
            }
        },
        "hash": {
            "url": "$url.sha256.txt"
        }
```

Reply:
ok, understand the "autoupdate url" change... makes it more clear and reliable....

Ok, thanks also for the 2nd change. I never understood the "hash url" magic with the "sha256.txt" extension as given here. Is there any explanations, docs, ... for this?

Reply:
No. It makes it work.

What magic? The URL will be used to search for hash to extract with generic regex.

https://scoop-docs.now.sh/docs/concepts/App-Manifest-Autoupdate.html#adding-hash-to-autoupdate